### ISSUE TITLE
Perhaps prune "AoC++" from scatterplot? Or normalize?

### ISSUE THREAD

**Issue by leijurv** [mevdschee/aoc-stats#4]
I donated to AOC, and now I show up as two different people: `leijurv` and `leijurv (AoC++)` when I hover :(

**Comment by mevdschee** [comment]
Yes, because this means I need to re-scrape this years data.. this is a limitation of the matching algorithm. I might do that by the end of the year.

**Comment by mevdschee** [comment]
I will leave this ticket open, as I have to do this again by the end of 2021
